### PR TITLE
Reorganize the repository structure

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Reorganized the repository structure to display on kyma-project.io

**Related issue(s)**
See also #320

### CODE REVIEW

File: guidelines/repository-guidelines/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Overview
+label: internal
+---
+
+The Repository guidelines include:
+
+- The [instruction](./01-new-repository-settings.md) that lists recommended settings for a new repository in the `kyma-project` and `kyma-incubator` organizations.
+- The content desription of the [`repository-template`](https://github.com/kyma-project/community/tree/master/guidelines/repository-guidelines/repository-template) folder that provides the structure of an exemplary Kyma repository. Use it whenever you create a new repository within Kyma.
```
Comment (kazydek) [inline guidelines/repository-guidelines/README.md]:
```suggestion
- The content description of the [`repository-template`](https://github.com/mmitoraj/community/tree/kyma-project-io/guidelines/repository-guidelines/repository-template) folder that provides the structure of an exemplary Kyma repository. Use it whenever you create a new repository within Kyma.
```

File: guidelines/technical-guidelines/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Overview
+---
+
+These guidelines provide rules and tips to all who contribute code to the Kyma repositories.
+
+Read about:
+
+- Rules for [naming](./01-naming.md) objects in Kyma.
```
Comment (kazydek) [inline guidelines/technical-guidelines/README.md]:
```suggestion
- Rules for [naming](./01-naming.md) objects in Kyma
```

File: guidelines/technical-guidelines/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- Different [component types](./02-kyma-components.md) in Kyma.
```
Comment (kazydek) [inline guidelines/technical-guidelines/README.md]:
```suggestion
- Different [component types](./02-kyma-components.md) in Kyma
```

File: guidelines/technical-guidelines/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+- Guidelines for writing [Custom Resource Definitions](./03-custom-resource-definitions.md) files.
```
Comment (kazydek) [inline guidelines/technical-guidelines/README.md]:
```suggestion
- Guidelines for writing [Custom Resource Definitions](./03-custom-resource-definitions.md) files
```

File: guidelines/technical-guidelines/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+- Tips and hints for providing [Docker images](./04-docker-images.md) in the context of Kyma.
```
Comment (kazydek) [inline guidelines/technical-guidelines/README.md]:
```suggestion
- Tips and hints for providing [Docker images](./04-docker-images.md) in Kyma
```

File: guidelines/technical-guidelines/README.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+- Best practices for creating [Helm charts](./05-helm.md).
```
Comment (kazydek) [inline guidelines/technical-guidelines/README.md]:
```suggestion
- Best practices for creating [Helm charts](./05-helm.md)
```

File: guidelines/technical-guidelines/README.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+- Guidelines for [defining APIs](./06-http-api-design.md) in Kyma.
```
Comment (kazydek) [inline guidelines/technical-guidelines/README.md]:
```suggestion
- Guidelines for [defining APIs](./06-http-api-design.md) in Kyma
```

File: guidelines/templates/README.md [hunk 8]
```diff
@@ -1,33 +1,33 @@
-# Templates
+---
+title: Overview
+---
 
-## Overview
-
-This directory contains templates for various document types in different Kyma repositories.
+In the [`community`](https://github.com/kyma-project/community/tree/master/guidelines/templates/resources) repository on GitHub under the `guidelines/templates/resources` folder you can find templates for various document types in different Kyma repositories.
```
Comment (kazydek) [inline guidelines/templates/README.md]:
```suggestion
In the [`community`](https://github.com/kyma-project/community/tree/master/guidelines/templates/resources) repository on GitHub under the `guidelines/templates/resources` folder, you can find templates for various document types to use in different Kyma repositories.
```

File: guidelines/technical-guidelines/02-kyma-components.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
+---
+title: Kyma components
+---
+
+The `components` directory contains the sources of all Kyma components.
```
Comment (kazydek) [inline guidelines/technical-guidelines/02-kyma-components.md]:
```suggestion
The [`components`](https://github.com/kyma-project/kyma/tree/master/components) directory contains the sources of all Kyma components.
```

File: README.md [hunk 10]
```diff
@@ -31,9 +31,9 @@ We are an open-source community and the number of contributors is growing. Meet
 
 ### Process
 
-Before you start to contribute, review the [CONTRIBUTING.md](CONTRIBUTING.md) document. To understand better the way the Kyma community works, read the [governance](governance.md) model and learn about the [Special Interest Groups and Working Groups](sig-and-wg/README.md) in Kyma.
+Before you start to contribute, review the [contributing.md](contributing/02-contributing.md) document. To understand better the way the Kyma community works, read the [governance](governance/governance.md) model and learn about the [Special Interest Groups and Working Groups](collaboration/README.md) in Kyma.
```
[the remark this answers is not in the document]
Comment (mmitoraj) [reply README.md]: It is actually`contributing.md`.

File: guidelines/repository-guidelines/README.md (hunk 1 above)
Comment (mmitoraj) [reply guidelines/repository-guidelines/README.md]: The link will be ok after the PR is merged.

File: guidelines/technical-guidelines/README.md (hunk 2 above)
Comment (mmitoraj) [reply guidelines/technical-guidelines/README.md]: Done for all bullet points

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -1,95 +1,4 @@
...
-
-Although it is the responsibility of the owner of the PR to ensure that the maintainers review and approve the PR, maintainers need to coordinate the overall number of unreviewed and unapproved PRs in their queue, and, if required, take appropriate measures to handle them effectively.
-
-To learn more about maintainers' responsibilities and rules for appointing new maintainers, and removing the existing ones, refer to the [governance.md](governance.md) document.
-
-## Owners
+## Overview
 
-To identify the owners of particular parts of your repository, see the `CODEOWNERS` file in the root directory.
+To contribute to this project, follow the rules from the `[contributing.md](/contributing/contributing.md)` document in the `[contributing](/contributing/) directory.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply CONTRIBUTING.md]: Actually, this comment is still valid for the incorrect formatting (``) but I can see now that the title of the doc is still `02-CONTRIBUTING.md` so the whole previous comment seems valid:

image

File: CODE_OF_CONDUCT.md [hunk 12]
```diff
@@ -1,13 +1,3 @@
...
-This document makes a reference to the rules of behavior for all contributors and maintainers in the community. All members agree to follow them to foster the growth of the community of tolerance, respect, and mutual understanding.
-
-## Code of conduct
-
-In the attempt to adopt the best practices from the most renowned open-source and cloud-native projects, and work closely with the Cloud Native Computing Foundation (CNCF), our community members undertake to follow the rules outlined in the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Read it to learn about the harassment-free environment, examples of unacceptable behavior, and the responsibilities of the project maintainers.
-
-## Violations
-
-If you observe any instance of abusive, harassing, or otherwise unacceptable behavior, report it as an issue to the project maintainers specified in the `CODEOWNERS` document.
+To to read about the Code of Conduct of this project, go to the [`CODE_OF_CONDUCT.md`](/contributing/CODE_OF_CONDUCT.md) document.
```
Comment (kazydek) [inline CODE_OF_CONDUCT.md]:
```suggestion
To read about the Code of Conduct of this project, go to the [`this`](/contributing/01-code-of-conduct.md) document.
```

File: README.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (kazydek) [reply README.md]: image

File: guidelines/templates/README.md [hunk 13]
```diff
@@ -1,33 +1,33 @@
...
+| [`community/sig-and-wg`](https://github.com/kyma-project/community/tree/master/sig-and-wg) | [`sig-wg-meeting-notes-template.md`](./resources/sig-wg-meeting-notes-template.md) | Use the template to document meeting notes of a given SIG or WG. | SIG and WG members |
+| [`website/content/blog-posts`](https://github.com/kyma-project/website/tree/master/content/blog-posts) | [`release-notes.md`](./resources/release-notes.md) | Use the template to write release notes. | Technical Writers |
+| Internal use | [`release-notes-input.md`](./resources/release-notes-input.md) | Use the template to provide input for the release notes. | Product Owners |
 
 
 >**NOTE:** Update the table each time you add a new template to the `resources` folder.
 
-See the [`internal-guidelines`](../../guidelines/internal-guidelines) folder for the template of an exemplary Kyma repository.
+See the [Repository guidelines](../repository-guidelines/) section for the template of an exemplary Kyma repository.
```
Comment (kazydek) [inline guidelines/templates/README.md]:
```suggestion
See the [repository guidelines](../repository-guidelines/) section for the template of an exemplary Kyma repository.
```

File: CONTRIBUTING.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (mmitoraj) [reply CONTRIBUTING.md]: I removed the 02-contributing.md and added it one more time - maybe this time Github will notice the name change.

File: CODE_OF_CONDUCT.md (hunk 12 above)
Comment (mmitoraj) [reply CODE_OF_CONDUCT.md]: Done, I just wouldn't put "this" in code.

File: CODE_OF_CONDUCT.md [hunk 14]
```diff
@@ -1,13 +1,3 @@
-# Community Code of Conduct
+# Overview
```
Comment (kazydek) [inline CODE_OF_CONDUCT.md]: As discussed, let's get rid of it and replace it with the doc title - **Conde of Conduct**.

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -1,95 +1,4 @@
...
-## Maintenance rules
-
-Every maintainer reviews each contribution according to the rules listed in this document.
-
-Although it is the responsibility of the owner of the PR to ensure that the maintainers review and approve the PR, maintainers need to coordinate the overall number of unreviewed and unapproved PRs in their queue, and, if required, take appropriate measures to handle them effectively.
-
-To learn more about maintainers' responsibilities and rules for appointing new maintainers, and removing the existing ones, refer to the [governance.md](governance.md) document.
-
-## Owners
+## Overview
```
Comment (kazydek) [inline CONTRIBUTING.md]: Same here, let's replace it with a proper doc title.

File: contributing/README.md [hunk 16]
```diff
@@ -0,0 +1,5 @@
+---
+title: Overview
+---
+
+Read the **Contributing** section to learn more about the Code of Conduct, contribution rules and workflows in Kyma
```
Comment (kazydek) [inline contributing/README.md]:
```suggestion
Read the **Contributing** section to learn more about the Code of Conduct, contribution rules, and workflows in Kyma.
```

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -1,95 +1,4 @@
...
 
-To identify the owners of particular parts of your repository, see the `CODEOWNERS` file in the root directory.
+To contribute to this project, follow the rules from the [this](/contributing/02-contributing.md)` document.
```
Comment (kazydek) [inline CONTRIBUTING.md]:
```suggestion
To contribute to this project, follow the rules from the [this](/contributing/02-contributing.md) document.
```

Comment (kazydek) [reply CONTRIBUTING.md]: And the contributing doc name ;)

File: README.md [hunk 18]
```diff
@@ -31,9 +31,9 @@ We are an open-source community and the number of contributors is growing. Meet
 
 ### Process
 
-Before you start to contribute, review the [CONTRIBUTING.md](CONTRIBUTING.md) document. To understand better the way the Kyma community works, read the [governance](governance.md) model and learn about the [Special Interest Groups and Working Groups](sig-and-wg/README.md) in Kyma.
+Before you start to contribute, review the [`contributing.md`](contributing/02-contributing.md) document. To understand better the way the Kyma community works, read about the [working model](governance/governance.md) and learn about the [Special Interest Groups and Working Groups](collaboration/README.md) in Kyma.
```
Comment (kazydek) [inline README.md]: contributing doc name ;)

File: README.md [hunk 19]
```diff
@@ -48,12 +48,12 @@ Feel invited to join the public [SIG Core meetings](https://kyma-community.slack
...
-- Document [templates](/guidelines/templates)
-- Overview of the [content strategy](guidelines/content-guidelines/content-strategy.md) and [content guidelines](/guidelines/content-guidelines)
-- [Release process](/guidelines/releases) details
-- [Decisions](/sig-and-wg/sig-core/decisions) made by Kyma team members and the SIG Core.
-- [Coding standards](/guidelines/coding-standards/README.md) 
+- [Naming](guidelines/technical-guidelines/01-naming.md) conventions
+- Document [templates](guidelines/templates)
+- Overview of the [content strategy](guidelines/content-guidelines/content-strategy.md) and [content guidelines](guidelines/content-guidelines)
+- [Release process](guidelines/releases-guidelines) details
+- [Decisions](collaboration/sig-core/decisions) made by Kyma team members and the SIG Core.
```
Comment (kazydek) [inline README.md]:
```suggestion
- [Decisions](collaboration/sig-core/decisions) made by Kyma team members and the Core SIG.
```

File: CODE_OF_CONDUCT.md (hunk 12 above)
Comment (kazydek) [reply CODE_OF_CONDUCT.md]: Yup, my bad.

File: introduction.md [hunk 20]
```diff
@@ -0,0 +1,57 @@
...
+
+Contact the Kyma team directly on [Slack](https://kyma-community.slack.com/messages/CBLBESMST/) to exchange ideas, ask questions, receive hints and tips, and extend your experience with Kyma.
+
+## Contribution & collaboration
+
+See how the Kyma team works and how to collaborate with other Kyma contributors. 
+
+### Contribution rules
+
+Review the [Code of Conduct](01-code-of-conduct.md) and understand the way the Kyma community works. Learn more about the [contributing process](02-contributing.md), the recommended [Git workflow](03-git-workflow.md) and [issues workflow](04-issues-workflow.md) that rely on forks, branches, rebasing, and squashing. Also, read about the [working model](../governance/governance.md) in the Kyma project.
```
Comment (kazydek) [inline introduction.md]: contributing file name

File: .kyma-project-io/introduction/introduction.md [hunk 21]
```diff
@@ -0,0 +1,90 @@
+---
```
Comment (magicmatatjahu) [inline .kyma-project-io/introduction/introduction.md]: Please change content of this file to:
```md
---
title: "Welcome to the Kyma community page!"
---

If you are looking for information on how to join us, you are in the right place. Read on to find out how you can get involved, contribute to Kyma code and documentation, and stay in touch with the latest Kyma news. Let's get started!

<div class="community-introduction">

<br />
<img class="community-introduction-first-img" align="left" src="assets/contribution-collaboration-icon.svg" height=100>
<p align="left">

# Contribution & Collaboration

</p>
<p>
<br />

## Contribution rules

Before you start contributing, read our [Code of Conduct](../contributing/01-code-of-conduct.md) and learn about our [working model](../governance/governance.md) to understand the way Kyma community works. Get familiar with the [contributing process](../contributing/02-contributing.md), recommended [Git workflow](../contributing/03-git-workflow.md), and [issues workflow](../contributing/04-issues-workflow.md) we use in Kyma so you can apply them later in practice as an active contributor.

## Guidelines

Follow guidelines for writing [content](https://github.com/kyma-project/community/tree/master/guidelines/content-guidelines) and [code](https://github.com/kyma-project/community/tree/master/guidelines/technical-guidelines). See how to create a [new repository](https://github.com/kyma-project/community/tree/master/guidelines/repository-guidelines) or a [new release](https://github.com/kyma-project/community/tree/master/guidelines/releases-guidelines), and use [templates](https://github.com/kyma-project/community/tree/master/guidelines/templates) for various document types.

## Bugs & feature request

Create an issue on [GitHub](https://github.com/kyma-project) to report a bug or request a feature. Go to the given repository's issue tracker and raise an issue using one of the available templates. The owners of the repository will review your issue and provide you with their feedback. If you are not certain which repository your issue refers to, open it in the [`kyma`](https://github.com/kyma-project/kyma) repository.

## Special Interest Groups and Working Groups

The Kyma community collaborates through Special Interest Groups and Working Groups ([SIGs and WGs](./collaboration/README.md)). By joining the SIGs and WGs, you can meet other community members, exchange information on Kyma, provide feedback, raise interest, ask questions, and learn about the product. Join the SIGs or WGs meeting to learn more what each group works on. 

</p>
<br />
<img  align="left" src="assets/technical-support-icon.svg" height=100>
<p align="left">

# Technical support

</p>
<p>
<br />

## Stack overflow

If you have a code-level question and need some expert advice, contact us on [Stack Overflow](https://stackoverflow.com/questions/tagged/kyma). Read through the existing questions tagged with `kyma` or ask your own if you don't find the answer you were looking for.

</p>
<br />
<img  align="left" src="assets/community-support-icon.svg" height=100>
<p align="left">

# Community support

</p>
<p>
<br />

## Slack

If you prefer a more direct contact, use one of the available [Slack channels](https://kyma-community.slack.com/messages/CBLBESMST/). Feel free to write to us with any questions, concerns, or ideas for improvement. Any constructive feedback or criticism is welcome.

</p>
<br />
<img  align="left" src="assets/stay-informed-icon.svg" height=100>
<p align="left">

# Stay informed

</p>
<p>
<br />

## Follow us on social media

We're on [Twitter](https://twitter.com/kymaproject) and [LinkedIn](https://www.linkedin.com/company/kyma-project/). Stay tuned for the news and spread the good word about Kyma among your followers.

## Watch us on YouTube

Subscribe to our [YouTube channel](https://www.youtube.com/channel/UC8Q8bBtYe9gQN-dQ-_L8JvQ) to see all recordings from SIGs and WGs meetings. You will also find there some other interesting videos, such as Kyma in non-English, Kyma Capabilities Demos, SIG Core meetings, and many more.

## Read our blog posts

Go the [Kyma blog](https://kyma-project.io/blog/) to read the latest news on our project, events, and teams. If you want to share your experience with Kyma, feel free to [write a blog post](https://github.com/kyma-project/website/blob/master/docs/write-blog-posts.md) on your own.

Thank you for your interest in Kyma. Hope you'll stay with us for a while.

</p>

</div>
```

File: guidelines/technical-guidelines/07-test-strategy.md [hunk 22]
```diff
@@ -92,4 +93,4 @@ Kyma test coverage is not complete yet, and probably will never be due to the na
 Because of the facts outlined above, some manual tests are always required before Kyma is released. Release candidates have to be verified manually following this process:
 1. Every time a missing test case is identified or a bug is found, an issue to cover that scenario with automated tests is created. The issue is labeled `test-missing` and assigned to the backlog of a team maintaining this area. In the case of unclear responsibility, one team is chosen. The issue description must contain a procedure on how to verify this scenario manually.
```
Comment (magicmatatjahu) [inline guidelines/technical-guidelines/07-test-strategy.md]: Please add blank line before list :)

File: .kyma-project-io/introduction/introduction.md [hunk 23]
```diff
@@ -0,0 +1,55 @@
...
+
+Subscribe to our [YouTube channel](https://www.youtube.com/channel/UC8Q8bBtYe9gQN-dQ-_L8JvQ) to see all recordings from SIGs and WGs meetings. You will also find there some other interesting videos, such as Kyma in non-English, Kyma Capabilities Demos, SIG Core meetings, and many more.
+
+## Blog
+
+Go the [Kyma blog](https://kyma-project.io/blog/) to read the latest news on our project, events, and teams. If you want to share your experience with Kyma, feel free to [write a blog post](https://github.com/kyma-project/website/blob/master/docs/write-blog-posts.md) on your own.
+
+Thank you for your interest in Kyma. Hope you'll stay with us for a while.
+
+</p>
```
Comment (magicmatatjahu) [inline .kyma-project-io/introduction/introduction.md]: Remove it

File: milv.config.yaml [hunk 24]
```diff
@@ -1,5 +1,5 @@
 white-list-external: ["localhost", "kyma.local", "kyma-system"]
-white-list-internal: ["mailto:kyma-security@googlegroups.com"]
+white-list-internal: ["mailto:kyma-security@googlegroups.com", "/.kyma-project-io/", "/collaboration/", "/contributing/", "/governance/", "/guidelines/content-guidelines/", "/guidelines/releases-guidelines/", "/guidelines/repository-guidelines/", "/guidelines/technical-guidelines/", "/guidelines/templates/README.md", "/guidelines/templates/templates-type.md"]
```
Comment (magicmatatjahu) [inline milv.config.yaml]: white-list-internal: ["mailto:kyma-security@googlegroups.com", "/community/"]

File: milv.config.yaml [hunk 25]
```diff
@@ -1,5 +1,5 @@
...
 black-list: ["/templates/","-template.md"]
```
Comment (magicmatatjahu) [inline milv.config.yaml]: black-list: ["/templates/","-template.md", "/.kyma-project-io/", "/collaboration/", "/contributing/", "/governance/", "/guidelines/content-guidelines/", "/guidelines/releases-guidelines/", "/guidelines/repository-guidelines/", "/guidelines/technical-guidelines/", "/guidelines/templates/README.md", "/guidelines/templates/templates-type.md"]

**Pull request merged**